comp/updater/bundle.go

```diff
@@ -0,0 +1,18 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package updater implements the updater component.
+package updater
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+// team: fleet
+
+// Bundle defines the fx options for this bundle.
+func Bundle() fxutil.BundleOptions {
+	return fxutil.Bundle()
```

Comment:
Should we add any of the updater components as part of the bundle?

Reply:
right now I think it's fine, but lmk if you see a need to use the bundle